cmd/storage/block/device-list.go

```diff
@@ -0,0 +1,50 @@
+package block
+
+import (
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/openebs/node-disk-manager/cmd/types/v1"
+)
+
+//ListBlockExec is for running os cmds for block disk and json parsing
+func ListBlockExec(resJsonDecoded *v1.BlockDeviceInfo) error {
+	//list block devices in json format
+	ListBlockCommand := v1.OsCommand{"lsblk", "-J"}
+	res, err := exec.Command(ListBlockCommand.Command, ListBlockCommand.Flag).Output()
+	if err != nil {
+		return err
+	}
+
+	//decode the json output
+	return json.Unmarshal(res, &resJsonDecoded)
+}
+
+//FormatOutputForUser is to print disk details to end user only with necessary fields
+func FormatOutputForUser(resJsonDecoded *v1.BlockDeviceInfo) {
```

Comment:
This looks like the usecase of go template!! Please raise an issue which handles this specifically.

Reply:
will do.